content/blog/2023-11-13.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "2023-11-13"
+date = 2023-11-13
++++
+
+---
+
+Today is Monday, day 317 of 365 (86.84%). We're in week 46 of 52 (88.46%).
+
+## Planning
+
+What I am going to do today:
+
+- [x] close [issue#29](https://github.com/OmnicodeSolutions/worklog-luisa/issues/29) from [Worklog](https://github.com/OmnicodeSolutions/worklog-luisa)
+- [ ] close [issue#5](https://github.com/OmnicodeSolutions/luisa_drf_flutter_client/issues/5) from [Flutter Client](https://github.com/OmnicodeSolutions/luisa_drf_flutter_client)
+- [x] close [issue#15](https://github.com/OmnicodeSolutions/luisa_drf_tutorial/issues/15) from [DRF Tutorial](https://github.com/OmnicodeSolutions/luisa_drf_tutorial)
+
+## Activities executed
+
+Today I worked on the three issues from the planning, but was not able to close the last one, I'm having troubles running flutter commands, I've tried using different flutter versions, stable channer and set a working directory, but nothing worked.
+
+I also did some tidying up in the [Flutter Client](https://github.com/OmnicodeSolutions/luisa_drf_flutter_client) (see Readings)
+
+## Learnings
+
+Today I learned about `extra_args` argument while making the [main.yml](https://github.com/OmnicodeSolutions/luisa_drf_tutorial/issues) to close [issue#15](https://github.com/OmnicodeSolutions/luisa_drf_tutorial/issues/15), it allowed me to pass `black --all-files` to pre-commit so that it would use the black linter in the whole project.
```

Comment:
Fix this `[main.yml](https://github.com/OmnicodeSolutions/luisa_drf_tutorial/issues)`. You should get a permalink to the file.